packages/components/organisms/f-offers/package.json

```diff
@@ -43,8 +43,10 @@
     "extends @justeat/browserslist-config-fozzie"
   ],
   "dependencies": {
+    "@justeat/f-globalisation": "1.0.0",
     "@justeat/f-services": "1.11.0",
-    "@justeat/f-globalisation": "1.0.0"
+    "axios": "0.21.1",
```

Comment:
Thought we were moving towards `f-http`? I'll read on to see if I can spot the reason...

Reply:
I have been informed that atm we are not production ready with f-http by @tommcclean